pages/02-01-up-and-running.md

```diff
@@ -22,15 +22,93 @@ Along with installing LÖVE, you will need a text editor for creating Lua files
 
 When you launch LÖVE, (see instructions below on how to do that) you will be greeted with a friendly graphic and the text "NO GAME", meaning you are running the engine but didn't give it a game to load.
 
-### macOS and other posix systems
+## macOS
```

Comment:
I think this should still be an h3 (`###`)  header to match the other sections.

Reply:
Fixed in latest commit.